mcal/src/ExternalInterrupt.cpp

```diff
@@ -54,19 +55,19 @@ void SetInterrupt(InterruptSource interrupt, InterruptMode mode, pFun callback){
     ExternalInterruptCallBacks[interrupt] = callback;
 }
 
-ISR(INT0) {
+ISR(1) {
```

Comment:
I think `INT0` is a macro that defined in `Atmega32.h`  https://github.com/abadr99/cpp_mcu_interface/blob/5a3fd046b5b3a71b1762056fb66ca77ee9814eb6/mcal/inc/Atmega32.h#L38

Is having the same value of magic number so Why do you think this change is necessary?

Reply:
I do not know why if i used 1 as it is it working , while if i use INT0 it does not work